.github/workflows/ci.yml

```
@@ -1,6 +1,6 @@
 name: Java CI
 
-on: [push, pull_request]
+on: [push]
```

```suggestion
on: [push, pull_request]
```

у тебя старая версия проекта, это строка должна выглядеть так

---

нужно исправить этот комментарий